### PR TITLE
Moved version to individual module properties

### LINKED ISSUE #56: Update ReadMe
With the merge of Location layer, we now offer an additional plugin. We also are close to merging to building plugin and the traffic plugin has been updated to 0.2.0. These changes need to be reflected in the readme file in this repo.

cc: @langsmith @zugaldia

### LINKED ISSUE #57: Make each plugin fully independent of each other
Currently the projects gradle.properties file contains the version number for all the projects. This needs to be split up into the different module gradle.properties file.

cc: @Guardiola31337 @zugaldia

### PR DESCRIPTION
**Pull request by cammace**
Closes #57 
Closes #56

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,8 +45,19 @@ These plugins are currently available:
 
 * **Traffic plugin:** Adds a real-time traffic layer to any Mapbox basemap.
 
+Dependency: `compile 'com.mapbox.mapboxsdk:mapbox-android-plugin-traffic:0.3.0'`
+
+
```
Comment (zugaldia) [inline README.md]: nit: extra line

File: plugins/building/gradle.properties [hunk 2]
```diff
@@ -1,3 +1,4 @@
+VERSION_NAME=0.1.0
```
Comment (zugaldia) [inline plugins/building/gradle.properties]: We should default to -SNAPSHOT versions on master.

File: plugins/locationlayer/gradle.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=0.1.0-SNAPSHOT
+VERSION_NAME=0.1.0
```
Comment (zugaldia) [inline plugins/locationlayer/gradle.properties]: We should default to -SNAPSHOT versions on master.

File: plugins/traffic/gradle.properties [hunk 4]
```diff
@@ -1,3 +1,4 @@
+VERSION_NAME=0.3.0
```
Comment (zugaldia) [inline plugins/traffic/gradle.properties]: We should default to -SNAPSHOT versions on master.

**Comment by cammace** [comment]
I've updated the version names for each plugin to use the correct snapshot versions. For the README, I've opened ticket https://github.com/mapbox/mapbox-plugins-android/issues/69 to track progress on splitting them up into the separate plugin directories.

**Pull request merged**